package.json

```diff
@@ -14,7 +14,10 @@
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@fontsource/roboto": "^5.0.5",
+    "@mui/icons-material": "^5.14.0",
     "@mui/material": "^5.13.7",
+    "@mui/x-date-pickers": "^6.10.0",
+    "dayjs": "^1.11.9",
```

Comment:
Do you have strong preferences using dayjs instead of date-fns?

Reply:
No, I can replace it